Rakefile

```diff
@@ -155,12 +156,14 @@ task :ci do
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:sinatra'
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:rack'
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:grape'
+    sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:graphql'
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:faraday'
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:aws'
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:mongodb'
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:sucker_punch'
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:dalli'
     sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:resque'
+    sh 'rvm $MRI_VERSIONS --verbose do appraisal contrib rake test:racecar'
```

Comment:
We evidently weren't running `racecar` tests in CI? Whoops.

Reply:
👍

Reply:
@delner if this PR should wait something else before the merge, remove this line and create a new PR so we can merge this fix immediately. Thank you!

Reply:
PR for this fix is here: https://github.com/DataDog/dd-trace-rb/pull/298